backend/tarsy/controllers/history_controller.py

```
@@ -254,6 +255,82 @@ async def get_session_summary(
+            session_id=session_id,
+            status=AlertSessionStatus(session.status)
+        )
+        
+    except HTTPException:
+        raise
+    except RuntimeError as e:
+        # Database unavailable - return 503
+        raise HTTPException(
+            status_code=503,
+            detail=f"History service unavailable: {str(e)}"
+        )
```

Ruff (0.14.5)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)